eng/pipelines/templates/jobs/ci.yml

```diff
@@ -197,6 +197,16 @@ jobs:
             -Artifacts ('${{ replace(convertToJson(parameters.ReleaseArtifacts), '''', '`''') }}' | ConvertFrom-Json | Where-Object -Not skipPublishPackage )
             -InformationAction Continue
 
+      - task: Powershell@2
+        inputs:
+          filePath: $(Build.SourcesDirectory)/eng/scripts/Save-Package-Namespaces-Property.ps1
+          arguments: >
+            -ArtifactStagingDirectory $(Build.ArtifactStagingDirectory)
+            -ArtifactsList ('$(ArtifactsJson)' | ConvertFrom-Json | Select-Object name, groupId, skipPublishDocMs | Where-Object -Not "skipPublishDocMs")
```

Comment:
I would suggest doing the filtering in the script

Reply:
Fixed with latest commit.